pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -57,6 +57,15 @@ func (alloc *Action) Execute(ssn *framework.Session) {
 	// jobsMap is used to find job with the highest priority in given queue.
 	jobsMap := map[api.QueueID]*util.PriorityQueue{}
 
+	alloc.prepareQueuesAndJobs(queues, jobsMap, ssn)
```

Comment:
We can rename it something like `pickUpQueuesAndJobs`

Reply:
good idea